### ISSUE TITLE
ini files being overwritten every time server starts after latest changes

### ISSUE THREAD

**Issue by rykugur** [PepeCitron/projectzomboid-server#8]
It looks like recent changes have caused my server's config files to be (mostly) overwritten. When connecting, my character is where I left him, but all of his gear is gone and all of the non-vanilla items are gone.

I've noted specifically that my `*Server.ini` file, the `Mods=` and `WorkshopItems=` lines have been set to empty (note: I was not using `MOD_IDS` and `MOD_NAMES` env variables, I had manually updated `*Server.ini` because some mod names have characters in them that would cause early string termination, and I don't know how to get around that). I've tried re-adding the mods/ids to these files, but they get overwritten every time the server starts.

I did try setting those env vars and ran into the same problem I detailed above, and now the server will not respond at all when trying to connect. `docker-compose config` looks correct, but when I examine the .ini files, the `Mods=` line is cut off due to a whitespace character in a mod name.

The `*_SandboxVars.lua` file has also been completely re-written with default values.

Sadly looks like `latest` is the only tagged release, so I can't roll back to the previous one.

**Comment by rykugur** [comment]
Looks like this change is indeed the culprit: https://github.com/PepeCitron/projectzomboid-server/blob/60bd13318c4be4c0a5d2209801c4da090a9cbe16/entry.sh

**Comment by rykugur** [comment]
Here's an exerpt from my Mods, it's falling over on `Diederiks Tile Palooza`

`Mods=FortKnoxLinked;EerieCountry;FortKnoxRoad;BedfordFalls;AddamsMansion;PertsPartyTiles;TryhonestyTiles;tkTiles_01;Diederiks Tile Palooza;DylansTiles;...`

Is there a way I can escape those whitespaces such that it doesn't truncate that string?

And the resulting `*Server.ini` file:

```
 cat data/config/Server/RollhaxPzServer.ini | grep -i Mods=
Mods=FortKnoxLinked;EerieCountry;FortKnoxRoad;BedfordFalls;AddamsMansion;PertsPartyTiles;TryhonestyTiles;tkTiles_01;Diederiks
```

**Comment by PepeCitron** [comment]
Fix in 4bccfc0

**Comment by rykugur** [comment]
Is that the correct commit?

**Comment by PepeCitron** [comment]
Ah yes I forgot to commit my changes 😑, it's fix in 69fb961b9146600e77483386914797a4ffc7b714

**Comment by rykugur** [comment]
Thank you for the quick turn around! I'll test this out this evening.
